### ISSUE TITLE
Tempest 4000 - graphics shifted up screen, or blank

### ISSUE THREAD

**Issue by tapperp** [yuzu-emu/yuzu#9117]
GAME: Tempest 4000
PC: Intel 1035G7 using Intel iGPU (GPD Win Max 2020)

In Yuzu build 1123 the graphics are drawn correctly using Vulkan. But in more recent builds the graphics are shifted upwards so that the bottom half of the screen is blank and only the lower part of the game is drawn at the top of the screen.

Note: in OpenGL no graphics are drawn in any builds, screen is just black.

I think the issue started sometime between build 1123 and 1150, but not sure exactly when.

Here is build 1123:
![01004120160be000_2022-10-23_01-35-04-272](https://user-images.githubusercontent.com/27729732/197375758-7733e334-0ab9-42c0-9fd5-9be4f21fb276.png)

Here is build 1207:
![01004120160be000_2022-10-23_01-37-12-643](https://user-images.githubusercontent.com/27729732/197375759-c311fa75-59ff-4aca-a6a3-98266bbed084.png)

**Comment by vonchenplus** [comment]
can not reproduce, please try the last EA

**Comment by tapperp** [comment]
> can not reproduce, please try the last EA

You are right. I tried on a different computer and the issue is not present. So, I think it may be related to either the specific hardware (1035G7 iGPU) and/or the latest graphics driver for it. It's not a Yuzu specific issue and probably doesn't effect most users. Oh well.

**Comment by Turdpeterson** [comment]
does on it steam deck as well
